ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -108,6 +108,12 @@ ${listenerText}
    * @type {string}
    */
   get installBoundaryText() {
+    if (this.features.isEnabled('byow-pki-hcp-cluster-id')) {
+      return `wget -q https://releases.hashicorp.com/boundary-worker/0.11.3+hcp/boundary-worker_0.11.3+hcp_linux_amd64.zip ;\\
+sudo apt-get update && sudo apt-get install unzip ;\\
+unzip *.zip ;\\
+boundary server -config="${this.configFilePath || '<path>'}/pki-worker.hcl"`;
```

Comment:
Does this work? Don't we need to install it to their PATH, or at the minimum call it from the directory?

Reply:
This is more or less just a suggestion and not intended to be the one true way to run a worker.  In production, administrators will almost certainly modify this script, or invoke in an entirely different way.